solutions/devsprint-vinicius-carvalho-3/DeliveryApp/Screens/Components/LoadingView.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  LoadingView.swift
+//  DeliveryApp
+//
+//  Created by Vitor Conceicao on 19/07/22.
+//
+
+import UIKit
+
+final class LoadingView: UIView {
+    private lazy var titleLabel: UILabel = {
+        let titleLabel = UILabel()
+        titleLabel.translatesAutoresizingMaskIntoConstraints = false
+        titleLabel.text = "Buscanado endereços..."
+        titleLabel.font = .systemFont(ofSize: 17, weight: .bold)
+        titleLabel.tintColor = .black
+        return titleLabel
+    }()
+    
+    private lazy var loadingIndicator: UIActivityIndicatorView = {
+        let loadingIndicator = UIActivityIndicatorView(style: .large)
+        loadingIndicator.translatesAutoresizingMaskIntoConstraints = false
+        return loadingIndicator
+    }()
+    
+    convenience init() {
+        self.init(frame: .zero)
+        setup()
+    }
+}
+
+extension LoadingView: ViewCode {
+    func setupSubviews() {
+        addSubview(titleLabel)
+        addSubview(loadingIndicator)
+    }
+    
+    func setupConstraints() {
+        setupTitleLabelConstraints()
+        setupLoadingIndicatorConstraints()
+    }
+    
+    func setupExtraConfiguration() {
+        loadingIndicator.startAnimating()
+    }
+    
+    private func setupTitleLabelConstraints() {
+        NSLayoutConstraint.activate([
+            titleLabel.bottomAnchor.constraint(equalTo: centerYAnchor, constant: -10),
```

Comment:
[Suggestion]
You can even leave it centered only in relation to centerY because the constraint of activate already has a distance of 10 in relation to it.